Makefile

```diff
@@ -171,40 +174,42 @@ install-completions:
 	install -m 755 -d ${DESTDIR}${BASHCOMPLETIONSDIR}
 	install -m 644 completions/bash/skopeo ${DESTDIR}${BASHCOMPLETIONSDIR}/skopeo
 
-shell: build-container
-	$(CONTAINER_RUN) bash
+shell:
+	$(CONTAINER_RUN) env SKOPEO_CONTAINER_TESTS=1 bash
 
 check: validate test-unit test-integration test-system
 
-# The tests can run out of entropy and block in containers, so replace /dev/random.
```

Comment:
I don't think it's necessary, at least in Cirrus-CI.  A few years ago I made contact with someone who (supposedly) convinced google to add in the option to turn on virtual RNG device in QEMU.  Additionally, we also have the "Make sure" solution of running the `rngd` service.  Otherwise yes, we had entropy problems before...to the extent that even the "login" getty service couldn't start :laughing:

In any case, short answer is "no", lack of entropy shouldn't be causing problems anymore.

Reply:
Great, thanks for the information (and having a written record here).